packages/wallets/bitkeep/src/adapter.ts

```diff
@@ -87,9 +78,9 @@ export class BitKeepWalletAdapter extends BaseMessageSignerWalletAdapter {
 
             this._connecting = true;
 
+            const adapter = new SolanaAdapter();
             // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-            const wallet = window.bitkeep!.solana!;
-
+            const wallet = adapter?.getProvider() || window.bitkeep!.solana!;
```

Comment:
New load-time dependencies cannot be introduced, they may only be loaded on connection. If `window.bitkeep` is not being used, the SDK must be asynchronously imported. I recommend checking for `window.bitkeep.solana` first, then `await import(...)` for the SDK. Please see some of the other adapters for how they do this.